src/utils/assets.ts

```diff
@@ -21,3 +22,12 @@ export const getToken = (
 
 export const isDefaultToken = (canisterId: string) =>
   !!Object.values(TOKENS).find(token => token.canisterId === canisterId);
+
+export const formatCollections = (collections: Collection[]) =>
```

Comment:
do we need this format? we could just use token.collection.description when needed. So we can keep some consistency with the ext